src/05-invoicing/invoices.apib

```diff
@@ -54,6 +55,7 @@ Get a list of invoices.
                 + paid: true (boolean)
                 + paid_at: `2016-02-04T16:44:33+00:00` (string, nullable)
                 + purchase_order_number: `000023` (string, nullable)
+                + payment_reference: `+++084/2613/66074+++` (string)
```

Comment:
Isn't it nullable as well?

Reply:
In the end, it shouldn't be anymore. We're filling them up and making sure they're always filled in. The `nullable` can be removed in a migration later.

Reply:
Oh, it has to stay `null` for draft invoices maybe?

Reply:
Indeed, forgot that. Will change it 👍